ClientApp/src/Pages/AdminSightingsList/AdminSightingsList.tsx

```diff
@@ -104,6 +113,15 @@ export function ListOfUnconfirmed(): JSX.Element {
 
         {unconfirmedSightingsData.sightings?.map((x) => <TableRow {...x} />)}
       </table>
+      <button
+            className="submit-button"
+            onClick={() => AdminFetchApiData()}
+            type="submit"
+            disabled={fetchClicked} aria-disabled={fetchClicked}
+          >
```

Comment:
bit confused where this '>' is coming from?

Reply:
Chloe, thats the closing of the button. i have moved to the end of prev line to avoid confusion